core/src/main/java/com/google/ar/sceneform/rendering/DepthTexture.java

```diff
@@ -0,0 +1,121 @@
+package com.google.ar.sceneform.rendering;
+
+import android.media.Image;
+import android.os.Handler;
+import android.os.Looper;
+
+import androidx.annotation.Nullable;
+
+import com.google.android.filament.Texture;
+import com.google.ar.sceneform.utilities.AndroidPreconditions;
+import com.google.ar.sceneform.utilities.BufferHelper;
+
+import java.nio.ByteBuffer;
+
+/**
+ * <pre>
+ *     The DepthTexture class holds a special Texture to store
+ *     information from a DepthImage to realize the occlusion of
```

Comment:
"implement the occlusion"?

Reply:
What you mean with "implement the occlusion"?

Reply:
realize -> implement

Reply:
done